tests/test_histograms.py

```diff
@@ -1,4 +1,4 @@
-from eventio.search_utils import collect_toplevel_of_type
+from eventio.search_utils import yield_toplevel_of_type
```

Comment:
I do not immediately get, why this test had to be modified in this PR ... it seems uncorrelated to the zcat matter ... but its no big deal either...

Reply:
zcat is not backwards seekable, this test made two passes through the data.

First collect all objects, that reading the contents of each object.